### PR TITLE
add tutorial for jiminy recommendation engine

### PR DESCRIPTION
**Pull request by elmiko**
This change adds the tutorial and related files for the recommendation engine

### CODE REVIEW

**Comment by elmiko** [comment]
@sophwats @ruivieira ptal

File: _applications/project-jiminy.adoc [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+   -e MODEL_STORE_URI=mongodb://mongo:mongo@mongodb/models \
+   -p APP_FILE=app.py \
+   -p APPLICATION_NAME=predictor
+....
+
+This will start the predictor and launch an Apache Spark cluster to run its
+workloads, it may take a minute or two for this process to complete. You can
+watch the build process by typing `oc logs -f bc/predictor`. Once the build
+is complete, you can ensure that the predictor is ready for use by watching
+it logs with the command `oc logs -f dc/predictor`, you will see this line
```
Comment (sophwats) [inline _applications/project-jiminy.adoc]: should read "watching its logs"

Comment (sophwats) [inline _applications/project-jiminy.adoc]: ". You will see this line..."

File: _applications/project-jiminy.adoc [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+oc new-app fabric8/s2i-java~https://github.com/radanalyticsio/jiminy-html-server.git \
+   -e SPRING_DATASOURCE_URL=jdbc:postgresql://postgresql:5432/postgres \
+   -e SPRING_DATASOURCE_USERNAME=postgres \
+   -e SPRING_DATASOURCE_PASSWORD=postgres \
+   -e OPENSHIFT_CONFIG_PREDICTOR_URL=http://predictor:8080/predictions/ranks \
+   --name html-server
+....
+
+This will command OpenShift to start building and then deploy the application.
+You can watch the build process by type `oc logs -f bc/html-server`. Once the
```
Comment (sophwats) [inline _applications/project-jiminy.adoc]: " by typing "

File: _applications/project-jiminy.adoc [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+....
+
+The output will look similar to this:
+
+....
+NAME          HOST/PORT                          PATH      SERVICES      PORT       TERMINATION   WILDCARD
+html-server   html-server-pj.shift.opb.studios             html-server   8778-tcp                 None
+....
+
+The value in the `HOST/PORT` column is what you will need, in this example you
```
Comment (sophwats) [inline _applications/project-jiminy.adoc]: two sentences: 
"is what you will need. In this..."

File: _applications/project-jiminy.adoc [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+would point your browser at `html-server-pj.shift.opb.studios`.
+
+Once you have opened your browser to the address specified by your deployment,
+you should see a page like this:
+
+pass:[<img src="/assets/project-jiminy/screenshot1.png" alt="Jiminy main page" class="img-responsive">]
+
+To see the predicted recommendations for a user, enter the user's ID value
+into the input labelled "Select User". This should be a numeric value. Don't
```
Comment (sophwats) [inline _applications/project-jiminy.adoc]: Maybe ?an integer value? ?a positive integer? ? a whole number? rather than "a numeric value".

**Comment by elmiko** [comment]
@sophwats thanks! fixed the changes you suggested

File: _applications/project-jiminy.adoc [hunk 5]
```diff
@@ -0,0 +1,344 @@
...
+sure to record the new values as you will need them to spawn the connected
+services.
+****
+
+=== Create application services
+
+With the storage services running, you can now deploy the application services
+which will perform the analytics and serve the user interface. _Now is a
+good time to ensure that you have performed the
+link:#prerequisites[prerequisite] oshinko install._
```
Comment (sophwats) [inline _applications/project-jiminy.adoc]: Is this link working? It sends me to a blank page. ?missing an 's' [prerequisites]

**Pull request merged**